<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>596</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Papyrus</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(173, 137, 255);
font-family: &quot;Papyrus&quot;</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>17</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="loginTextEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Enter thy name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="confirmTextEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,50);</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string> Was that your name? Enter Y or N</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="lcdClockLabel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
